### PR TITLE
Update file-4.x connector doc

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
[Update file-4.x connector doc](https://github.com/wso2/docs-mi/commit/2e748d3da96852bf68f3aa295552a1f77dad8eec)

### CODE REVIEW

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 1]
```diff
@@ -4,261 +4,189 @@ File Connector can be used to perform operations in the local file system as wel
...
+* `/read `: The user sends the request payload, which includes the location of the file that needs to be read. This request is sent to the integration runtime where the FileConnector API resides. Once the API is invoked, it first checks if the file exists in the specified location. If it exists, the content is read and response is sent to the user. If the file does not exist, it sends an error response to the user.
 
-Create a folder in your local file system with read and write access. This will be your working directory. In this example, it is `/Users/hasitha/temp/file-connector-test/dataCollection`. 
+    <img src="{{base_path}}/assets/img/integrate/connectors/filecon-3.x/fileconnector-02.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
 
-!!! Note
-    If you set up a FTP server, SFTP server, or a Samba server, do the required configurations and select a working directory. Save the host, port, and security related information for future use. 
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
 ## Configure the connector in WSO2 Integration Studio
```
Comment (gabilang) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:21]:
```suggestion
## Setting Up the Integration Project
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 2]
```diff
@@ -4,261 +4,189 @@ File Connector can be used to perform operations in the local file system as wel
...
 
-Follow these steps to set up the Integration Project and the Connector Exporter. 
+Follow these steps to set up the Integration Project and the Connector Exporter Project.
```
Comment (gabilang) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:23]:
```suggestion
Follow these steps to set up the Integration Project using the WSO2 Micro Integrator Visual Studio Code extension.
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 3]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
 
 ## What you'll build
 
-This example describes how to use the File Connector to write messages to local files and then read the files. Similarly, the same example can be configured to communicate with a remote file system (i.e FTP server) easily. The example also uses some other WSO2 mediators to manipulate messages. 
+This example explains how to use File Connector to create a file in the local file system and read the particular file. The user sends a payload specifying which content is to be written to the file. Based on that content, a file is created in the specified location. Then the content of the file can be read as an HTTP response by invoking the other API resource upon the existence of the file.
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:7]: Let's keep 'Similarly, the same example can be configured to communicate with a remote file system (i.e FTP server) easily. The example also uses some other WSO2 mediators to manipulate messages. '

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 4]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
-
-    1.  Add the <b>checkExist</b> operation of the File connector to the canvas.
-    2.  Create a new file connection pointing to the working directory we already set up. Keep this as the File connection for the operation.  
+    <img src="{{base_path}}/assets/img/integrate/connectors/filecon-3.x/fileconnector-02.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
 
-        <img src="{{base_path}}/assets/img/integrate/connectors/filecon10.png" title="working directory" width="800" alt="working directory"/>
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-    3.  Configure the file path as `/dataCollection/employees/employees.csv`. This file will store the employee information. 
+## Setting Up the Integration Project
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:21]:
```suggestion
## Set up the integration project
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 5]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
 
-        !!! Note
-            Be sure to append a new line at the end of the file. The <b>Write Mode</b> needs to be `Create New`. 
+4. You have now created the new project with the following structure.
 
-        <img src="{{base_path}}/assets/img/integrate/connectors/filecon13.png" title="create new" width="800" alt="create new"/>
+     <img src="{{base_path}}/assets/img/integrate/connectors/prject-structure.png" title="Adding a Rest API" width="500" alt="Adding a Rest API"/>
 
-16. After the Filter mediator, use the Enrich mediator again to put back the saved payload into the message payload. 
+### Creating the Integration Logic
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:39]:
```suggestion
### Create the integration logic
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 6]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
 
-    <img src="{{base_path}}/assets/img/integrate/connectors/filecon14.png" title="enrich - put back payload" width="800" alt="enrich - put back payload"/>
+#### Creating the First Resource
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:41]:
```suggestion
#### Create the first resource
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 7]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
+                    <enableLock>false</enableLock>
+                    <updateLastModified>true</updateLastModified>
+                    <includeResultTo>Message Body</includeResultTo>
+                </file.write>
+            </inSequence>
+            <faultSequence>
+            </faultSequence>
+        </resource>
+    ```
+#### Creating the Second Resource
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:90]:
```suggestion
#### Create the second resource
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 8]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
 
-??? note "Click here for instructions on deploying on WSO2 Enterprise Integrator 6"
-    1. You can copy the composite application to the `<PRODUCT-HOME>/repository/deployment/server/carbonapps` folder and start the server.
-
-    2. WSO2 EI server starts and you can login to the Management Console https://localhost:9443/carbon/ URL. Provide login credentials. The default credentials will be admin/admin. 
-
-    3. You can see that the API is deployed under the API section. 
+2.  Click on **Select Destination** to select a destination folder to export the CAR file.
 
 ## Testing
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:156]:
```suggestion
## Test
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 9]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
 
-1. Create a file called data.json with the following payload. 
+### File Create Operation
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:158]:
```suggestion
### File create operation
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 10]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
+**Expected Response**:
 
-    !!! Info
-        Curl Application can be downloaded from [here](https://curl.haxx.se/download.html).
+You should get a 'Success' response, and the file should be created in the specified location in the above payload.
 
-    ```bash
-    curl -H "Content-Type: application/xml" --request POST --data @body.json http://10.100.5.136:8290/fileconnector/create
-    ```
+### File Read Operation
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:174]:
```suggestion
### File read operation
```

File: en/docs/reference/connectors/file-connector/file-connector-example.md [hunk 11]
```diff
@@ -4,261 +4,192 @@ File Connector can be used to perform operations in the local file system as wel
...
 
-4.  If you invoke the API again with a different set of employees, the new employees will get appended to the same file. The response you receive will include all the employees that were added from both messages.
+You should get the following text returned.
 
-In this example, the File connector was used to create a file, write to a file, and to read a file. By blending these capabilities together with other powerful message manipulation features of WSO2, it is possible to define a working scenario in minutes. The File connector has many more functionalities. Refer the [File Connector reference guide]({{base_path}}/reference/connectors/file-connector/file-connector-config/) for more information. 
+```
+This is a test file.
+```
 
 ## What's Next
```
Comment (chanikag) [inline en/docs/reference/connectors/file-connector/file-connector-example.md:193]:
```suggestion
## What's next
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**